mixer/pkg/runtime/handler/table_test.go

```diff
@@ -108,6 +110,38 @@ func TestNew_NoReuse_DifferentConfig(t *testing.T) {
 	}
 }
 
+func TestNew_NoReuse_DifferentConnectionConfig(t *testing.T) {
+	templates := map[string]*template.Info{}
+	adapters := map[string]*adapter.Info{}
+
+	// Load base dynamic config, which includes listentry template and listbackend adapter config
+	dynamicConfig, err := data.ReadConfigs("../../../template/listentry/template.yaml", "../../../test/listbackend/nosession.yaml")
+	dynamicConfig = data.JoinConfigs(dynamicConfig, data.InstanceDynamic, data.RuleDynamic)
+
+	// Join base dynamic config with dynamic handler
+	config1 := data.JoinConfigs(dynamicConfig, data.ListHandler3)
+	if err != nil {
+		t.Fatalf("Fail to load dynamic config: %v", err)
+	}
+	s, _ := config.GetSnapshotForTest(templates, adapters, data.ServiceConfig, config1)
+	table := NewTable(Empty(), s, nil)
+
+	// Join base dynamic config with dynamic handler which has different connection address
+	config2 := data.JoinConfigs(dynamicConfig, data.ListHandler3Addr)
+	// NewTable again using the slightly different config
+	s, _ = config.GetSnapshotForTest(templates, adapters, data.ServiceConfig, config2)
+
+	table2 := NewTable(table, s, nil)
+
+	if len(table2.entries) != 1 {
+		t.Fatal("size")
```

Comment:
can you write something more descriptive than `size`?

Reply:
Done.